packages/expo-file-system/ios/FileSystemExceptions.swift

```diff
@@ -97,3 +97,9 @@ final class FailedToFindAssetException: GenericException<String> {
     "Failed to find photo library asset: \(param)"
   }
 }
+
+final class CannotGetSharedContainer: GenericException<String> {
+  override var reason: String {
+    "Unable to open shared container with provided app group \(param). Check your entitlements"
```

Comment:
`Check your entitlements`– could we make this more descriptive, like letting folks know which entitlements are missing or an FYI link?

Reply:
_all file-system changes are reverted_